src/test/6_MainRegistry.t.sol

```diff
@@ -1038,3 +1038,142 @@ contract PricingLogicTest is MainRegistryTest {
         assertEq(expectedLiquidationThreshold, actualLiquidationThreshold);
     }
 }
+
+contract DelegateTest is MainRegistryTest {
```

Comment:
Add tests under ASSET MANAGEMENT in the testfile please, we are testing batchProcessWithdrawal and batchProcessDeposit

Reply:
we are actually testing an authorization modifier that by chance is on the batchprocess functions

Reply:
No we are testing the functions batchProcessWithdrawal and batchProcessDeposit that have a modifier, we do it the same way for all other modifiers such as onlyOwner, onlyVault etc.